cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+---
+CPS: 10
+Title: Wallet Connectors
+Status: Open
+Category: Wallets
+Authors:
+    - Ryan Williams <ryan.williams@iohk.io>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/619
+Created: 2023-07-28
+---
+
+## Abstract
+Wallets are a foundational element of web 3, being the primary interface between users and blockchains.
+Wallet-web bridges allow users to connect their wallet to web-based stacks, allowing a wide range of specialized experiences.
+
+Wallet web-bridge standards consist of two parts; connection standard and API.
+With the connection standard defining how the wallet and dApp initiate communication, this could be over injected Javascript object, WebRTC, WebTorrent, etc.
+The API defines what communications look like between the dApp and wallet post connection.
+
+This problem statement is concerned with the issues surrounding Cardano's current and future wallet-web bridges.
+These interfaces are difficult to define and historically have been even harder to iterate upon.
+We wish to provide a comprehensive catalog of the current offerings; and their drawbacks to be able to make suggestions on future standards.
+
+### Acknowledgments
+
+<details>
+  <summary><strong>First workshop - 2023-11-27</strong></summary>
+
+  I would like to thank those that contributed to the first workshop hosted by Adam Dean and Ryan Williams ([see presentation slides with notes](https://docs.google.com/presentation/d/1lXw1yMf-Hyp1eVfaFnbTV9897r6pZEd6zpvLNMqT3cI/)).
+  - Beatrice Anihiri
+  - Denis Kalinin
+  - Evgenii Lisitskii
+  - George APEX Pool
+  - George Flerovsky
+  - George Humphreys
+  - Hernán Rajchert
+  - Jack Rousian
+  - Joshua Marchand
+  - Ken Fritschy
+  - Ken-Erik Ølmheim
+  - Leo H
+  - Marcel Baumberg
+  - Martynas Kazlaukas
+  - Michael Chappell
+  - Mircea Hasegan
+  - Nicolas Ayotte
+  - Rhys Bartels-Waller
+  - Robert Phair
+  - Rodolpho Ribeiro
+  - Steven Johnson
+  - Teodora Sevastru Lunn
+  - Thomas Lindseth
+  - Thomas Upfield
+  - Vladimir Kalnitsky
+
+</details>
+
+
+## Problem
+<!-- A more elaborate description of the problem and its context. This section should explain what motivates the writing of the CPS document. -->
+
+The core motivation of this document is to catalog the current state of Cardano's wallet connectors, discuss their flaws and identify keys concerns for future connector authors to be aware of.
+We hope that by discussing the issues we will inspire the next generation of Cardano wallet connectors so that the ecosystem can grow beyond it's first iterations.
+
+Ineffective dApp-connectors can cause a range of issues for wallets, dApp developers and thus users.
+Due to the nature of these connections there can often be unforeseen impacts of small design decisions.
+
+For users, connectors should be widely supported across platforms, secure and decentralized.
```

Comment:
> decentralized

I'm yet to see a single decentralized wallet. Isn't the purpose quite the opposite - lower decentralization for convenience (i.e. alleviate the need to run own node)?

I would phrase it differently: users want to use existing dApps reliably and are in general interested in having more dApps rather than less.